sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_models.hpp

```diff
@@ -394,7 +394,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
   };
 
   /**
-   * @brief Content type of the certificate when downloaded from getecret.
+   * @brief Content type of the certificate when downloaded from getsecret.
```

Comment:
Should this be `GetSecret` instead of `getsecret`?

Reply:
Yep, ideally it should.

Reply:
Looks like this doc comment was incorrectly copied from .NET:
https://github.com/Azure/azure-sdk-for-net/blob/cb3e8630ab0df6609b5ff6643e5ea0700e4a9816/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateContentType.cs#L10

https://github.com/Azure/azure-sdk-for-net/pull/7530

Here's the corresponding swagger:
https://github.com/Azure/azure-rest-api-specs/blob/3e27c70e7c02c07b458bc0e94716c3d82d3fdd19/specification/keyvault/data-plane/Microsoft.KeyVault/stable/7.3/certificates.json#L1387
>           "description": "The content type of the secret. eg. 'application/x-pem-file' or 'application/x-pkcs12', "

@schaabs and @vhvb1989 what is the correct/intended documentation here?

Reply:
Filed an issue for this: https://github.com/Azure/azure-sdk-for-cpp/issues/4394